docs/learn/smart-contract-developers/mint-lsp7-token.md

```diff
@@ -8,196 +8,89 @@ import TabItem from '@theme/TabItem';
 
 # Mint LSP7 Token
 
-:::caution Disclaimer
+In this guide you will mint some [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) tokens to the EOA that controls the contract.
```

Comment:
```suggestion
This guide will teach you how to mint [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) tokens as an EOA contract owner.
```
"Controls" might be misleading in this context. I would suggest updating the sentence for clarity.

Reply:
Applied the second part, left the first part as it is. I remember discussing with Fabian about not using "teaching"